packages/ocs/dashboards/persistent-internal/details-card.tsx

```diff
@@ -85,6 +89,14 @@ const DetailsCard: React.FC = () => {
           >
             {serviceVersion}
           </DetailItem>
+          <DetailItem
+            key="inTransitEncryption"
+            title={t('In-transit Encryption')}
```

Comment:
```suggestion
            title={t('In-transit encryption')}
```
Please check this with UX team